__test__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CreateCustomerForm } from '@/components/form/CustomerForm';
+import sinon, { spy } from 'sinon';
+import { Customer } from '@/lib/types/apiTypes';
+import { ApiUpdateResponse } from '@/lib/types/hooks';
+
+const customerEndpointURL = `${process.env.NEXT_PUBLIC_API_URL}/customer`;
+const bodySpy = sinon.spy((body) => body);
+const pathSpy = sinon.spy((path) => path);
+
+jest.mock('@/lib/utils/fetch', () => ({
+    post: async (path: string, body: object): Promise<ApiUpdateResponse<Customer>> => pathSpy(path) && bodySpy(body),
```

Comment:
Here we *mock*  the function `post` defined in `@/lib/utils/fetch`. What it means is that when anything attempts to call `post` from `@/lib/utils/fetch`, it calls this instead.

So what happens in this hook

https://github.com/three-consulting/epoc-frontend/blob/390347c6e069404141393b21ffca3792e1ce518c/lib/hooks/useCustomers.ts#L24-L28

is that the `post` in that hook is replaced by this mock function we define here.

The point is that, in this test, we only care about the fact that `post` has been called. Not about the actual thing that `post` does.